modulefiles/module_base.orion

```diff
@@ -1,52 +1,41 @@
-#%Module######################################################################
-##
-##      FV3GFS prerequisites
-##
+#%Module################################################
+# FV3GFS prerequisites
+########################################################
 
 module use /apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack
 module load hpc/1.1.0
 module load hpc-intel/2018.4
-module load hpc-impi/2018.4
-
-#module load hpss/hpss
-module load nco/4.8.1
-module load gempak/7.5.1
 
-module load grib_util/1.2.2
-module load prod_util/1.2.2
+# gsi
+module unload mkl/2020.2
+module load mkl/2018.4
 
-module load crtm/2.3.0
-setenv CRTM_FIX /apps/contrib/NCEPLIBS/orion/fix/crtm_v2.3.0
+module load hpc-impi/2018.4
+module load slurm/19.05.3-2
 
-module load jasper/2.0.25
-module load zlib/1.2.11
-module load png/1.6.35
+# waveprep
+module load cdo/1.9.5
```

Comment:
This module can only be loaded after hdf5 and netcdf are loaded.

Please do not move the module loads around.
The order of module loads is critical.

Reply:
Orion-login-1[1] /home/lgan$ module purge
Orion-login-1[2] /home/lgan$ module use /apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack
Orion-login-1[3] /home/lgan$ module load hpc/1.1.0
Orion-login-1[4] /home/lgan$ module load hpc-intel/2018.4
Orion-login-1[5] /home/lgan$ module unload mkl/2020.2
Orion-login-1[6] /home/lgan$ module load mkl/2018.4
Orion-login-1[7] /home/lgan$
Orion-login-1[7] /home/lgan$ module load hpc-impi/2018.4
Orion-login-1[8] /home/lgan$ module load slurm/19.05.3-2
Orion-login-1[9] /home/lgan$ module load cdo/1.9.5
Orion-login-1[10] /home/lgan$ module list

Currently Loaded Modules:
  1) hpc/1.1.0   2) intel/2018.4   3) hpc-intel/2018.4   4) mkl/2018.4   5) impi/2018.4   6) hpc-impi/2018.4   7) munge/0.5.13   8) slurm/19.05.3-2   9) hdf5/1.10.6  10) netcdf/4.7.4  11) cdo/1.9.5

This shows that the hdf5 and netcdf is loaded after cdo module.

Reply:
Answered in Files changed page.

Reply:
In a global-workflow PR's meeting on 12/8 3:45p, Rahul requested to module load netcdf and hdf5 before module load cdo. Keep module load cdo on the bottom of this file.